x/observer/keeper/tss.go

```diff
@@ -66,6 +66,7 @@ func (k Keeper) RemoveTSS(ctx sdk.Context) {
 	store.Delete([]byte{0})
 }
 
+// TODO define a type for a list of TSS isntead of using []types.TSS
```

Comment:
Could you elaborate on this?

Reply:
Removed that todo , we dont need it any more